client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -16,29 +16,17 @@
         class="c-public-statement__header"
         :class="{'border--bottom': isOpen}">
         <div class="layout__item u-11-of-12 u-1-of-4-desk-up align-sub u-pl-0">
-          <div class="inline-block u-mr-0_5">
-            <input
-              v-if="showCheckbox"
-              type="checkbox"
-              :id="number"
-              name="item_check[]"
-              :value="id">
-            <label
-              :for="number"
-              data-cy="statementNumber"
-              class="inline u-mb-0 u-ml-0_25">{{ number || '' }}</label>
-          </div><!--
-       --><div class="inline-block">
+        <div class="inline-block">
```

Comment:
Removing this affects several templates, and at least in the "Meine Entwürfe" view I'm sure that we want to keep the checkbox.